spec/features/card_spec.rb

```diff
@@ -28,7 +28,7 @@
       visit new_card_path
       card_fill_in
       select2_fill_in '.card_deck_name', with: 'Test'
-      click_button 'Создать'
+      click_button 'Создать карточку'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.